File: Uri/1040.py
x = raw_input()
x = x.split(' ')
n1 = float(x[0])
n2 = float(x[1])
n3 = float(x[2])
n4 = float(x[3])

med = (n1*2 + n2*3 + n3*4 + n4)/10
print"Media: %.1f"%med

if med >= 7:
    print"Aluno aprovado."
elif med < 5:
    print"Aluno reprovado."
else:
    print"Aluno em exame."
    x = float(raw_input())
    print"Nota do exame: %.1f"%x
    med = (med+x)/2
    if med >= 5:
        print"Aluno aprovado."
    else:
        print"Aluno reprovado."
    print"Media final: %.1f"%med

File: Uri/1040.c
#include<stdio.h>

int main(){
    float n1, n2, n3, n4, av, ne, ar;
    scanf("%f %f %f %f", &n1, &n2, &n3, &n4);
    av = ((n1*2)+(n2*3)+(n3*4)+(n4*1))/10;
    printf("Media: %.1f\n",av);
    if (av >= 7){
        printf("Aluno aprovado.\n");
    }
    else if(av < 5){
        printf("Aluno reprovado.\n");
    }
    else{
        printf("Aluno em exame.\n");
        scanf("%f", &ne);
        printf("Nota do exame: ");
        ar = (ne + av)/2;
        if(ar >= 5){
            printf("Aluno aprovado.\n");
        }
        else{
            printf("Aluno reprovado.\n");
        }
        printf("Media final: %.1f\n", ar);
    }
    return 0;
}
